package.json

```
@@ -226,6 +246,19 @@
           "when": "resourceLangId == markdown",
           "group": "navigation@30"
         }
+      ],
+      "notebook/toolbar": [
+        {
+          "command": "runme.runCategory",
+          "when": "notebookType == runme && notebookHasCategories",
+          "group": "navigation"
+        }
+      ],
+      "notebook/cell/execute": [
+        {
+          "command": "runme.runCellCategory",
+          "group": "navigation"
```

```suggestion
          "group": "navigation",
          "when": "notebookType == runme && notebookHasCategories"
```

Need to include the same when clause - otherwise it shows up in other notebook types.

---

Fixed